static/app/views/performance/landing/widgets/components/widgetContainer.tsx

```diff
@@ -244,15 +243,18 @@ export const WidgetInteractiveTitle = ({
       options={menuOptions}
       value={chartSetting}
       onChange={handleChange}
-      renderWrapAs={TextOverflow}
```

Comment:
This isn't necessary to achieve text overflow ellipsis. All we need to do is add `min-width: 0` to `StyledCompactSelect` (see below).

**Before:**
<img width="406" alt="Screenshot 2023-01-11 at 3 08 13 PM" src="https://user-images.githubusercontent.com/44172267/211937494-a45d33a5-4efe-4514-9c59-3caf9dc771a8.png">

**After: text is still ellipsized**
<img width="406" alt="Screenshot 2023-01-11 at 3 09 12 PM" src="https://user-images.githubusercontent.com/44172267/211937678-25a72274-02d2-4327-b527-4e4dced2eb1a.png">